src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,16 @@
+.background {
+  background: #0091ea;
+}
+
+.textColor {
+  color: #fff0f0;
+}
+
+.alignmentRow1 {
+  text-align: left;
+  padding-left: 4vw;
```

Comment:
don't use the unit `vw` with padding, sometimes the view is big or small and mostly you want a specific padding not a changed one depending on the screen

Reply:
Okay, changes are done, check it now please.